features/admins/admin_invitations.feature

```diff
@@ -309,10 +309,12 @@ Feature: Admin Actions to Manage Invitations
       And I fill in "Username" with "dax"
       And I press "Search" within "form.invitation.simple.search"
     Then I should see "Copy and use"
+    Then I should see "Delete"
```

Comment:
Similar to the other feature test, please update the repeated step types to use `And`, for example
```suggestion
      And I should see "Delete"
```